### DISCUSSION TITLE
Integration of multi-objective frameworks

### DISCUSSION THREAD

**Discussion by AmelieLiese** [openml/automlbenchmark#530] [General]
Hi, is it possible to integrate AutoML frameworks that optimize for multiple criteria?

**Comment by PGijsbers** [comment]
It is possible to do the integration, but officially there is only single objective support. If you define a task with multiple metrics, you can still see them in the `TaskConfig`, so you can use it fine as far as I am aware. But the official usage for it would be "optimize towards the first, and only report on the others".

 I did start working towards official support for multi-objective optimization in #531.
